tests/conformance_suite.postman_collection.json

```diff
@@ -4229,6 +4228,54 @@
 					"script": {
 						"type": "text/javascript",
 						"exec": [
+							"// Raw body should exclude all optional items and should contain",
+							"// first alternate version of any 'oneOf' elements defined in the",
+							"// OpenAPI schema.",
+							"const date = new Date();",
+							"const isoDate = date.toISOString();",
+							"pm.environment.set('currentDate', isoDate.substring(0, isoDate.length - 5) + 'Z');",
+							"",
+							"pm.variables.set(\"rawBody\", {",
+							"    \"credential\": {",
+							"        \"@context\": [",
+							"            \"https://www.w3.org/2018/credentials/v1\"",
+							"        ],",
+							"        \"type\": [",
+							"            \"VerifiableCredential\"",
+							"        ],",
+							"        \"issuer\": \"{{credential_issuer_id}}\",",
+							"        \"issuanceDate\": \"{{currentDate}}\",",
+							"        \"credentialSubject\": \"{{credential_subject}}\"",
+							"    },",
+							"    \"options\": {",
+							"        \"type\": \"Ed25519Signature2018\",",
+							"        \"created\": \"{{currenDate}}\"",
+							"    }",
+							"});",
+							"",
+							"// Obtain an access token with the required \"issue:credentials\" scope",
+							"if (!pm.variables.get('currentAccessToken')) {",
+							"    utils(pm).getAccessToken('issue:credentials', (err, res) => {",
+							"        pm.expect(err).to.be.null;",
+							"        pm.variables.set('currentAccessToken', res.json().access_token)",
+							"    });",
+							"}",
+							"",
+							"// Obtain the credential issuer ID via did:web resolution",
+							"if (!pm.variables.get('credential_issuer_id')) {",
+							"    // TODO: This needs to be populated correctly",
+							"    pm.variables.set('credential_issuer_id', 'temporary.placeholder')",
+							"}",
+							"",
+							"// Set the credential subject to use for testing",
+							"if (!pm.variables.get('credential_subject')) {",
+							"    pm.variables.set('credential_subject', 'did:example:123')",
+							"}",
+							"",
+							"// Set the credential id to use for testing",
+							"if (!pm.variables.get('credential_id')) {",
+							"    pm.variables.set('credential_id', \"urn:uuid:{{$randomUUID}}\")",
+							"}",
```

Comment:
Current conformance testing for `presentations/verify` does not have a request body, did you intend to make this change in another request?

```suggestion
```